packages/realm-app-importer/src/AppImporter.ts

```diff
@@ -136,6 +150,25 @@ export class AppImporter {
     return { appId };
   }
 
+  public async deleteApp(clientAppId: string): Promise<void> {
+    await this.logIn();
```

Comment:
Hmm. I just noticed we're doing this in `importApp` as well. It was written originally as a once pr process thing, but with the `realm-app-importer serve`, we shouldn't log-in again and again. I've [created an issue](https://github.com/realm/realm-js/issues/4280) to tackle this.

Reply:
👍